portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,16 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random fun fact to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFunFact() {
+  const funFacts =
+      ['I am a Capricorn sun, Aquarius Moon, and Cancer rising', 'I am fluent in Spanish and English', 'I adopted a dog when I was 15, and his name is Austin!', 'I am a first-generation college student.'];
 
```

Comment:
similar nit: Put each fact on its own line. If you can set up TSLint in your editor I recommend it, that'll do some automatic formatting for you